### ISSUE TITLE
MD Simulations with increased atom counts

### ISSUE THREAD

**Issue by lastconnor123** [libAtoms/QUIP#583]
Hello QUIP package managers,

I'd like to be able to train a GAP potential using a dataset of around 100 atoms and then simulate NVT and NPT ensembles containing a few thousand atoms in a larger cell using the fitted potential. Is this possible?

**Comment by gabor1** [comment]
Yes of course, that's how almost everyone uses it. You'll need to ensure that your small training cells cover a range of densities.

**Comment by lastconnor123** [comment]
Brilliant. The training set does cover a range of densities. I just need to know what parameters I need to change to increase the number of simulated atoms. For reference, I'm using the GAP tutorial found in this deepnote:

https://deepnote.com/workspace/ioan-bogdan-magdau-00195eee-2451-4f21-a901-fac6c0d22b2c/project/GAP-liquid-solvent-tutorial-ca2046f3-c62f-4f79-92cb-f1198bcebf68/notebook/notebook-0fad9c7b042442a685d779c770a01c64

**Comment by gabor1** [comment]
I'm pretty sure I don't understand your problem. Have you simply tried to evaluate the GAP model on a structure which contains many atoms?

**Comment by gabor1** [comment]
You don't need to change any parameters of the GAP model

**Comment by lastconnor123** [comment]
I would like to use the trained GAP model (where the training set contains
only around 100 atoms in a 13x13x13 Angstrom cell) to simulate a larger
environment how a few thousand atoms in a larger cell (5x5x5 nm). We have a
successful GAP model, now we would like to use it in larger NVT and NPT
simulations. We do not know how to do that

**Comment by gabor1** [comment]
My question to you remains the same.have you actually tried evaluating the GAP model on a large structure with thousands of atoms? It will just work. Nothing to be adjusted. 

Out of curiosity, what MD program are you planning to use? Lammps? ASE? Quip itself ?I recommend lammps for large systems, it has good parallelism. You need to compile QUIP in serial and lammps in parallel.

**Comment by gabor1** [comment]
If on the other hand you don't know how to create an Atoms object in ASE with many atoms, I suggest that you study the ASE system a little bit - you will need this anyway if you are going to use it: https://wiki.fysik.dtu.dk/ase . If you want better parallel performance, then study lammps, and how it us used to initialise structures and run GAP. Again you'll need to learn your way around lammps input files to be effective. https://www.lammps.org/
